kubernetes/deployment.yml

```diff
@@ -23,7 +23,13 @@ spec:
             path: config.yml
       containers:
       - name: leanix-exporter
-        image: giantswarm/leanix-exporter:latest
+        image: giantswarm/leanix-exporter:{{SHA1}}
```

Comment:
This should be `[[ .SHA ]]`. Architect templates it using the square brackets to not conflict with the Helm templates. See https://github.com/giantswarm/architect/blob/a822bc15ddaaf589a16a522b052a5bb1f1a68b62/template/helm.go#L75.